src/sentry/api/endpoints/organization_has_mobile_app_events.py

```diff
@@ -43,8 +43,8 @@ def _get(self, request, organization):
             # TODO: update discover query for null browser and client_os_name is android or ios
             result = discover.query(
                 query=query,
-                orderby="browser.name",
-                selected_columns=["browser.name", "project", "id", "client_os.name"],
+                orderby="-timestamp",
+                selected_columns=["browser.name", "client_os.name", "timestamp"],
```

Comment:
Do you need to select the timestamp? It doesn't seem to be used.

Reply:
@lynnagara I got an error saying I can't order by a field that isn't being selected

Reply:
Ok, must be a restriction of the discover module